docs/deploy-coe-with-Kafka.md

```diff
@@ -1,6 +1,6 @@
 # Citrix ADC Observability Exporter with Kafka as endpoint
 
-Citrix ADC Observability Exporter is a container that collects metrics and transactions from Citrix ADC. It also transforms the data into the formats (such as AVRO) that are supported in Kafka and exports the data to the endpoint. Kafka is an open-source and distributed event streaming platform for high-performance data pipelines and streaming analytics.
+Citrix ADC Observability Exporter is a container that collects metrics and transactions from Citrix ADC. It also transforms the data into the formats (such as AVRO) that is supported in Kafka and exports the data to the endpoint. Kafka is an open-source and distributed event streaming platform for high-performance data pipelines and streaming analytics.
```

Comment:
that 'are' seems fine i think?

Reply:
The auto spell check was suggesting it. However, felt are in more accurate. so changed it.